src/libtorch.cc

```diff
@@ -852,11 +933,30 @@ ModelInstanceState::Execute(
     // enable/disable inference mode - supersedes NoGradGuard
     torch::InferenceMode infer_guard(model_state_->EnabledInferenceMode());
 
-    // NvFuser is only supported for GPU instances. No change is made if
-    // enable_nvfuser is not set by user
-    auto nvfuser_pair = model_state_->EnabledNvfuserPair();
-    if (std::get<0>(nvfuser_pair)) {
-      if (std::get<1>(nvfuser_pair) && (device_ != torch::kCPU)) {
+    // JIT. No change is made unless parameter is explicitly set.
+    if (std::get<0>(model_state_->EnabledJitProfiling())) {
+      torch::jit::getProfilingMode() =
+          std::get<1>(model_state_->EnabledJitProfiling());
+    }
+
+    if (std::get<0>(model_state_->EnabledJitExecutor())) {
+      torch::jit::getExecutorMode() =
+          std::get<1>(model_state_->EnabledJitExecutor());
+    }
+
+    // Fuser. Parameter is ignored if NVFuser parameter is explicitily
```

Comment:
Are all these fuser options supported on CPU as well? If not we will need to add checks for device like we do for NV-Fuser.